packages/ckeditor5-ckbox/src/ckboxediting.ts

```diff
@@ -68,6 +70,8 @@ export default class CKBoxEditing extends Plugin {
 		if ( isLibraryLoaded() ) {
 			editor.commands.add( 'ckbox', new CKBoxCommand( editor ) );
 		}
+
+		verifyUploadAssetsPermission( editor );
```

Comment:
This is an `async` function, but called without handling of the returned promise. There can be errors during asynchronous execution. They should be handled here. Or there should be a comment describing that unhandled promise warning on the browser's console is intended (is it?).

Reply:
I would rather stay with comment option, because this function is asynchronous only because of `httpRequest` which already have error handling, logic which left could be our usual synchronous function which in regular situation didn't require any error handling.

Reply:
But to be honest I have no idea for comment, or eventually how to handle this `Promise`. Maybe there are some suggestions?

Reply:
Maybe `verifyUploadAssetsPermission` is too long (doing too much) and should be shortened a bit?

For example we could change the name to `isUploadPermissionGranted` and returns `true` or `false`. And depend on the result block the uploads commands (or not).

WDYT?